src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -82,6 +82,11 @@
           "actual size could be less depending on elements type")           \
           range(0, max_jint)                                                \
                                                                             \
+  develop(intx, SuperWordMaxVectorSize, 64,                                 \
```

Comment:
The flag can't be develop because it is used in product code. It should be `diagnostic`.